docs/data/material/varnish/varnish-with-react-router/varnish-with-react-router.md

```diff
@@ -0,0 +1,18 @@
+---
+title: varnish-with-react-router
+githubLabel: 'varnish: varnish-with-react-router'
+---
+
+# Varnish With React Router
+
+<p class="description">AI2 Varnish With React Router Usage</p>
+
+This is documentation for how you can use React Router in a Varnish App and inherit Varnish theming for links.
+
+[React Router](https://reactrouter.com/en/main) is a tool commonly used for client-side routing. You can pull this into your Varnish App and by default get the Varnish theming for react-router links. Below is an example of how this can look when routing to pages and getting a scroll-to-top experience upon navigating to a page.
+
+The Getting Started link shows how to employ the contrast colors of Varnish links via the linkContrast class if you are using a dark theme. 
```

Comment:
does it? either way, please repeat it here....

Reply:
should have been more clear in the wording here but this refers to the "Getting Started" link in this markdown file, directly below this line of code, so it is highlighted here :) I'll add the word "below" to make this more apparent. 

<img width="842" alt="Screen Shot 2023-03-14 at 11 18 31 AM" src="https://user-images.githubusercontent.com/6945628/225100557-79703699-e827-4c6b-8048-8b2a785dcbfa.png">